vivarium/core/emitter.py

```diff
@@ -34,15 +34,11 @@
 MONGO_DOCUMENT_LIMIT = 1e7
 
 HISTORY_INDEXES = [
-    'time',
-    'type',
-    'simulation_id',
+    'data.time',
```

Comment:
Interesting that `time` is no longer an index. Did you test that `data.time` lets you retrieve the nested field?

Reply:
Just tested it on my local machine and it works great. Here is the [relevant documentation](https://www.mongodb.com/docs/manual/core/index-single/#create-an-index-on-an-embedded-field).

Reply:
LGTM